pyproject.toml

```diff
@@ -44,7 +44,9 @@ line-length = 120
 profile = "black"
 
 [tool.pyright]
-exclude = ["venv"]
 ignore = ["test*.py", "examples/*.py", "docs/*"]
+pythonVersion = "3.10"
 typeCheckingMode = "strict"
 reportPrivateUsage = false
+reportPropertyTypeMismatch = "warning"
+reportUnnecessaryTypeIgnoreComment = "warning"
```

Comment:
Since the ruleset is strict, this is unnecessary.

Reply:
These two rules are off even in strict, to be fair, but that's valid. I can remove them.